HM2_task3.py

```diff
@@ -0,0 +1,15 @@
+# 3
+
+casino_blacklist = ["Jack Pie", "Erik Man", "Lili Evans"]
+poker_blacklist = ["Patrick Low", "Ann Downstown", "Mari Darret"]
+alcohol_blacklist = ["Albert Second", "Kate Signi", "Iri Mane"]
+
+casino_blacklist.extend(alcohol_blacklist)
+casino_blacklist.extend(alcohol_blacklist)
+print(casino_blacklist)
```

Comment:
It is not a solution. You just add all lists together. Please read task more cerefully 
![image](https://user-images.githubusercontent.com/25909810/214870121-79f9bc62-4367-432d-b485-6047889be7f5.png)